src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,59 @@
 package core.basesyntax;
 
-public class WorkWithFile {
+import java.io.*;
```

Comment:
common mistake 
<img width="517" alt="image" src="https://user-images.githubusercontent.com/72727656/193845409-bd412bf3-9de5-4bc0-8d6f-a481269e6a4e.png">

Reply:
I saw it, but couldnt fix. Thanks for help)